mypy-baseline.txt

```diff
@@ -599,6 +601,9 @@ posthog/api/organization_feature_flag.py:0: error: Invalid index type "str | Non
 posthog/api/organization_feature_flag.py:0: error: Invalid index type "str | None" for "dict[str, int]"; expected type "str"  [index]
 posthog/api/organization_feature_flag.py:0: error: Invalid index type "str | None" for "dict[str, int]"; expected type "str"  [index]
 posthog/api/notebook.py:0: error: Incompatible types in assignment (expression has type "int", variable has type "str | None")  [assignment]
+posthog/year_in_posthog/year_in_posthog.py:0: error: Item "None" of "dict[Any, Any] | None" has no attribute "get"  [union-attr]
+posthog/year_in_posthog/year_in_posthog.py:0: error: Argument 1 to "stats_for_user" has incompatible type "dict[Any, Any] | None"; expected "dict[Any, Any]"  [arg-type]
+posthog/year_in_posthog/year_in_posthog.py:0: error: Item "None" of "dict[Any, Any] | None" has no attribute "get"  [union-attr]
```

Comment:
These are already "handled" here and don't want to fix them https://github.com/PostHog/posthog/blob/ff2271bf879dad6b4fbe200c3695ec63d35e948b/posthog/year_in_posthog/year_in_posthog.py#L130-L135